### PR TITLE
23222 comment out 1pass udpate section by 20241001

### PR DESCRIPTION
**Pull request by kzdev420**
*Issue #:* /bcgov/entity#23222

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: legal-api/Makefile [hunk 1]
```diff
@@ -115,16 +115,19 @@ push: #build ## Push the docker container to the registry & tag latest
...
-	oc -n "$(OPS_REPOSITORY)-$(TAG_NAME)" exec "dc/vault-service-$(TAG_NAME)" -- ./scripts/1pass.sh \
-		-m "secret" \
-		-e "$(TAG_NAME)" \
-		-a "$(DOCKER_NAME)-$(TAG_NAME)" \
-		-n "$(OPENSHIFT_REPOSITORY)-$(TAG_NAME)" \
-		-v "$(VAULTS)" \
-		-r "true" \
-		-f "false"
+
+# 1Password CLI1 will be deprecated on Oct 1, 2024
```
Comment (vysakh-menon-aot) [inline legal-api/Makefile:119]: Only in legal-api? what about rest of the makefile in lear?

Comment (chenhongjing) [reply legal-api/Makefile:119]: Think it's ok to verify the changes for one component and then modify the others. How do you think?

Comment (vysakh-menon-aot) [reply legal-api/Makefile:119]: you need to remove these call
image

Comment (vysakh-menon-aot) [reply legal-api/Makefile:119]: > Think it's ok to verify the changes for one component and then modify the others. How do you think?

I am ok with it

Comment (chenhongjing) [reply legal-api/Makefile:119]: @kzdev420  FYI, in case you didn't see Vysakh's comment

Comment (kzdev420) [reply legal-api/Makefile:121]: Hi @chenhongjing @vysakh-menon-aot 

Updated.

**Review by chenhongjing — APPROVED** [review]

**Review by JazzarKarim — APPROVED** [review]

**Pull request merged**